src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -248,9 +255,11 @@ export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
     const numOfPeopleWithNoRoles = this.orgPersonList.filter(people => people.roles.length === 0).length
 
     if (this.entityFilter(EntityTypes.BCOMP)) {
-      return numOfCompletingParty === 1 && numOfIncorporator >= 1 && numOfDirector >= 1 && numOfPeopleWithNoRoles === 0
+      return numOfCompletingParty === 1 && numOfIncorporator >= 1 && numOfDirector >= this.getMinimumDirectorCount &&
+        numOfPeopleWithNoRoles === 0
     } else if (this.entityFilter(EntityTypes.COOP)) {
-      return numOfCompletingParty === 1 && numOfIncorporator >= 3 && numOfDirector >= 3 && numOfPeopleWithNoRoles === 0
+      return numOfCompletingParty === 1 && numOfIncorporator >= 3 && numOfDirector >= this.getMinimumDirectorCount &&
+        numOfPeopleWithNoRoles === 0
```

Comment:
I am in favour of these rules being here as this component is (for now) specific to BEN/ULC/etc businesses.

I am thinking that we would have a different version of this component for future, significantly-different businesses.

For now this is good 👍